README.md

```diff
@@ -1,94 +1,87 @@
-# The ops library
+Ops is a Python framework for developing and testing charms. It is an official component of the Charm SDK, itself a crucial part of the Juju universe.
+||||
+|-|-|- |
+|| Learn about Juju! | Start deploying charms: <br> https://juju.is/docs/juju <br> _Juju is declarative, model-driven, and has native support for integrations and scaling, on any cloud. You want to deploy `foo`? Run `juju deploy foo`! You wish to integrate `foo` and `bar`? Run `juju integrate foo bar`!_ |
+||||
+|| Get better at Juju! | Take advantage of the charm ecosystem: <br> https://charmhub.io/ <br> _Quickly deploy a cluster ([OpenStack](https://charmhub.io/openstack-base), [Kubernetes](https://charmhub.io/charmed-kubernetes)), a database ([PostgreSQL](https://charmhub.io/postgresql-k8s), [MongoDB](https://charmhub.io/mongodb), etc.), an observability stack ([Canonical Observability Stack](https://charmhub.io/cos-lite)), an MLOps solution ([Kubeflow](https://charmhub.io/kubeflow)), and so much more!_ |
+||||
+|:point_right:| **Make Juju your own!** | Write your own charm: <br> https://juju.is/docs/sdk <br> _Juju is written in Go, but our SDK supports easy charm development in Python!_ |
 
-<!-- The text below is also at the top of ops/__init__.py. Keep in sync! -->
+## Use Ops
+Give it a quick try:
 
-The ops library is a Python framework ([`available on PyPI`](https://pypi.org/project/ops/)) for developing
-and testing [Juju](https://juju.is/) charms in a consistent way, using standard Python constructs
-to allow for clean, maintainable, and reusable code.
```

Comment:
I think it's important to surface the clean, maintainable, reusable goals here.

Reply:
I'm ambivalent on this one -- I think that stuff kind of goes without saying.